apps/variables.tf

```diff
@@ -62,15 +62,15 @@ variable "transform_apps_ad_group_display_name" {
   type = string
 }
 
-variable "transform_developers_ad_group_display_name" {
+variable "transform_apps_ad_group_principal_id" {
```

Comment:
named correctly? seems inconsistent with the others

Reply:
Yes, as it comes from transform and also is consistent with other group related variables.
What would you recommend/name?

Reply:
oh the AD groups for apps is created in transform? fair enough! I would have expected in in apps/ but 🤷🏼

Reply:
Can move that in apps. Will do that in a separate PR.